schema.graphql

```diff
@@ -1,68 +1,77 @@
 
 type Module {
-    """
-    CoinGecko API V3
-    """
+  """
+  CoinGecko API V3
+  """
+
+  ########################################################################
+  #                                 PING                                 #
+  ########################################################################
   
-  # """
-  # Check API server status
-  # """
   ping: Ping!
+  """ Check API server status """
   
-  # simplePrice(
-  #   """
-  #   Get the current price of any cryptocurrencies in any other supported currencies that you need.
-  #   """
-  #   ids: String
-  #   vs_currencies: String
-  #   include_market_cap: String
-  #   include_24hr_vol: String
-  #   include_24hr_change: String
-  #   include_last_updated_at: String
-  # ): Map! @annotate(type: "Map<String!, SimplePrice!>!")
-  
-  # """
-  # Get current price of tokens (using contract addresses) for a given platform in any other currency that you need.
-  # """
-  # simpleTokenPrice(
-  #   id: String
-  #   contract_addresses: String
-  #   vs_currencies: String
-  #   include_market_cap: String
-  #   include_24hr_vol: String
-  #   include_24hr_change: String
-  #   include_last_updated_at: String
-  # ): Map! @annotate(type: "Map<String!, SimplePrice!>!")
+  ########################################################################
+  #                                SIMPLE                                #
+  ########################################################################
+
+  simplePrice(
+    """
+    Get the current price of any cryptocurrencies in any other supported currencies that you need.
+    """
+    ids: String
+    vs_currencies: String
+    include_market_cap: Boolean
+    include_24hr_vol: Boolean
+    include_24hr_change: Boolean
+    include_last_updated_at: Boolean
+  ): Map! @annotate(type: "Map<String!, Map<String!, BigNumber>!>!")
+  
+  simpleTokenPrice(
+    """
+    Get current price of tokens (using contract addresses) for a given platform in any other currency that you need.
+    """
+    id: String!
+    contract_addresses: String
+    vs_currencies: String
+    include_market_cap: Boolean
+    include_24hr_vol: Boolean
+    include_24hr_change: Boolean
+    include_last_updated_at: Boolean
+  ): Map! @annotate(type: "Map<String!, Map<String!, BigNumber>!>!")
   
-  """
-  Get list of supported_vs_currencies.
-  """
+
   simpleSupportedVsCurrencies: [String!]!
-  
+  """ Get list of supported_vs_currencies. """
+
+  ########################################################################
+  #                                COINS                                 #
+  ########################################################################
+
   coinsList(
       """
       List all supported coins id, name and symbol (no pagination required)
       """
     include_platform: Boolean
   ): [CoinListItem!]!
-  
-  """
-  List all supported coins price, market cap, volume, and market related data
-  """
+
   coinsMarkets(
```

Comment:
Not sure if it matters, but docgen won't read these comments correctly. It assumes comments precede the item they refer to. So docgen will think the function comment here is a comment on the `vs_currency` property.

```graphql  
coinsMarkets(
    """
    List all supported coins price, market cap, volume, and market related data
    """
    vs_currency: String!
    ids: String
    category: String
   ...
  ): [CoinMarketItem!]!
```

For docgen to work, the comment has to be like this:
```graphql  
"""
List all supported coins price, market cap, volume, and market related data
"""
coinsMarkets(
    vs_currency: String!
    ids: String
    category: String
   ...
  ): [CoinMarketItem!]!
```

Reply:
Ok, will refactor it.